Environmental/new_crawlers/Weather forecast/anta_weather_forecast/producer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import requests
+import datetime
+from kafka import KafkaProducer
+from dotenv import load_dotenv
+
+ANTA_LAT = 36.963974
+ANTA_LON = 30.727117
+ANTA_RESOLUTION = "6km"
+FROM_DATE = datetime.datetime.now().strftime("%Y-%m-%d")
+TO_DATE = (datetime.datetime.now() + datetime.timedelta(days=7)).strftime("%Y-%m-%d")  # TO_DATE = FROM_DATE + 7 days
+VARIABLES = ["temperature2m", "rh2m", "windspeed10m", "precipitation"]
+TIMEZONE = "Europe/Athens"
+
+
+def get_seven_day_forecast():
+    params = {
+        "lat": ANTA_LAT,
+        "lon": ANTA_LON,
+        "resolution": ANTA_RESOLUTION,
+        "from_date": FROM_DATE,
+        "to_date": TO_DATE,
+        "variables": ','.join(VARIABLES),
+        # "timezone": TIMEZONE,
+        "apikey": os.environ["API_KEY"]
+    }
+    url = os.environ["DRAXIS_API"] + os.environ["FORECAST_URI"]
+    response = requests.get(url=url, params=params)
+    return response.json()
+
+
+def broadcast_to_topic(topic, forecast):
```

Comment:
this method seems quite redundant. unless we plan on doing something more with it, i would say remove it and use `producer.send(topic, forecast)` directly